Dockerfile.arm64

```diff
@@ -1,14 +1,12 @@
-FROM arm64v8/debian:9.8-slim
-
-LABEL maintainer "Casey Davenport <casey@tigera.io>"
+FROM scratch
 
 ADD licenses/ /licenses
```

Comment:
this won't work will it?

Reply:
Oh d'oh - sorry this IS a local copy.  My bad!

Reply:
So in that case we seem to have a mixture of using local license file and copying in from ubi.  I really think we should at least converge on one method otherwise it raises questions about why they are different.

Reply:
I pinged @lmm  regarding this. I locally verified cluster comes up with that. Once @lmm  acks RH validation is still OK with such image, I will update this PR.